etc/schema.json

```diff
@@ -1069,10 +1072,16 @@
                         "type": "object",
                         "properties": {
                             "rdata": {
-                                "type": "string"
+                                "type": "string",
+                                "suricata": {
+                                    "keywords": ["dns.response.rrname"]
+                                }
                             },
                             "rrname": {
-                                "type": "string"
+                                "type": "string",
+                                "suricata": {
+                                    "keywords": ["dns.answers.rrname", "dns.response.rrname"]
+                                }
```

Comment:
Had previously discussed something like:
```json
"suricata": {
    "keyword": "primary-keyword",
    "related_keywords": ["related1", "related0"]
}
```
But I really want to keep it simple, otherwise its not likely to get done.

Reply:
wow i didn't know we were going to update the entire schema to reflect keywords. I thought we'll try to find approximate matches by calculating word weight or something.
So, this will have to be a manual update to the entire schema?

Reply:
Yes, this was the idea from the start. Have it be an authoritative source of these relationships.